src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -228,6 +226,40 @@ private void PerformSecurityChecks()
             {
                 HasSuspiciousContent = true;
             }
+
+            // A local function to check if the ScriptBlockAst is essentially a safe HashtableAst.
```

Comment:
Is it 'essentially' a hash table or is it in fact a hash table?  Is there a difference?

Reply:
I think a better comment is "Can skip code scan if the file is a psd1 file representing a safe hash table.", or to that effect.

Reply:
I will update the comment. Thanks!